arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -806,6 +806,38 @@ def rule_gfm_1_3_17(
             )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_1_3_18(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 1.3.18] The submitter's taxonomy contains an element whose name matches an element in the EDINET taxonomy.
+    Modify the element name so that it does not match the EDINET taxonomy, or use the element in the EDINET taxonomy and delete the
+    element added in the submitter's taxonomy.
+    """
+    for extensionConcept in pluginData.getExtensionConcepts(val.modelXbrl):
+        name = extensionConcept.get("name")
+        if name is None:
+            name = ""
+        concepts = val.modelXbrl.nameConcepts.get(name, [])
+        for concept in concepts:
+            if not pluginData.isExtensionUri(concept.qname.namespaceURI, val.modelXbrl):
```

Comment:
We want to check the file URI here, not the namespace URI (I see that this is issue pre-exists elsewhere)